### DISCUSSION TITLE
[Projects beta] Show labels in kanban view

### DISCUSSION THREAD

**Discussion by rhys-vdw** [community/community#10788] [Projects and Issues] [answered]
Would be nice to be able to see labels on each issue card (like Trello).

**Comment by cbarina** [comment, marked as answer]
You can? You can customize what's shown on a card via the view's dropdown --> Configuration --> first option (select Labels)

![image](https://user-images.githubusercontent.com/7785375/151930258-8145b3e9-53d5-4920-80d4-627691da0c25.png)

**Reply by jordanpattern** [reply]
This is good to know, but I never would have discovered it on my own.

**Reply by rhys-vdw** [reply]
Haha, thanks. Yeah I also didn't realize. I guess I'd suggest it's on by default.

**Reply by darcydriscoll** [reply]
Agreed, I didn't realise either. Personally I don't see the benefit of not keeping it on by default. Maybe this is a broader issue of the layout options not being visible/explained enough in the UI?

**Reply by adamgadomski** [reply]
thanks for that hint !

**Reply by scott-lin** [reply]
Just wanted to add to the feedback train -- I too was looking for a way to view labels in the board view, and this configuration option did not stand out to me for getting what I wanted.

Thanks for the tip.

**Reply by mrobraven** [reply]
Any way to turn this on permanently for a project? It's getting very irritating having to re-do this every time the page refreshes

**Reply by mrobraven** [reply]
> Any way to turn this on permanently for a project? It's getting very irritating having to re-do this every time the page refreshes

Oh how embarrassing, ignore me. I've just realised there's a big 'Save' button that I wasn't clicking🤦

**Reply by wyckster** [reply]
This makes sense but was difficult to discover.  I suggest that labels should be a visible field **by default**.

**Reply by jeder** [reply]
@mrobraven -- for what it's worth, I stepped on the same issue... I think we are getting spoiled by HTML 5 and auto-saving settings pages ;-)

**Reply by mattcantstop** [reply]
I looked for this forever and could not find this! I just wanted to add labels! I looked in settings multiple times thinking it'd be in there.

**Reply by jasondainter** [reply]
Thanks this was killing me...

**Reply by Hinthusiast** [reply]
That took me a while! Thank you!

**Comment by thclark** [comment]
~~This used to be present in the old projects, ant it'd be great to have it back. At present we've got an absolute mass of text-based cards on the board, it'd be good to get some labels in there - the distribution of colour gives you an instinctive sense on the focus of work upcoming on your board, meaning you're able to spot resource distribution problems and reprioritise.~~

~~The labels could also be a convenient UX tweak - click on a label to add a filter for it saves you typing the filter.~~

Yup, all of this is already there *embarrased shuffle out of the room*

**Comment by Mochitto** [comment]
Stumbled into this issue for the same reason as other people. I feel like the UI doesn't make this easily discoverable? 
Thanks for the answer :)

**Comment by srojevsky** [comment]
> Any way to turn this on permanently for a project? It's getting very irritating having to re-do this every time the page refreshes

@mrobraven you can save the view after you changed that and it is going to stay.
